### DISCUSSION TITLE
Confused about loading models.

### DISCUSSION THREAD

**Discussion by vdruts** [oobabooga/text-generation-webui#480] [General]
Getting nothing but errors. Do models have to be .pt only or can they be .bin? Can models go into separate folders? or do the files have to be in the root /model folder?

**Comment by BetaDoggo** [comment]
Most models need to be placed in their own folder alongside any of their helper files. Here's the structure for Pygmalion as an example:

Inside the folder `models/Pygmalion-2.7B/`
```
config.json
merges.txt
pytorch_model.bin
special_tokens_map.json
tokenizer_config.json
vocab.json 
```
The main exception to this is 4bit models which have their `.pt` file placed in the root of the models folder but still require the folder of the original version to be setup in the way shown above(everything except the .bin files).

**Comment by vdruts** [comment]
Thanks. Does the folder require any specific naming convention? I guess I'm
confused I got a slew of errors even on the original models placed into
folders.

**Reply by BetaDoggo** [reply]
The default naming scheme on most models downloaded from huggingface should work. What models are you trying to use and what errors are you getting?

**Reply by vdruts** [reply]
This shows the errors.

https://github.com/oobabooga/text-generation-webui/issues/479

**Reply by smokeyhallow** [reply]
Shouldn't this be documented somewhere? I was confused looking at the model repos
